index.js

```diff
@@ -149,11 +141,33 @@ function createExtensionMap(exts) {
     return null;
   }
 
-  if (isEmpty(exts)) {
+  if (!Object.keys(exts).length) {
     return { '': null };
   }
 
   return exts;
 }
 
+function isString(value) {
+  if (typeof value === 'string') {
+    return true;
+  }
+
+  if (Object.prototype.toString.call(value) === '[object String]') {
+    return true;
+  }
+
+  return false;
+}
+
+function defaultsNotNullish(mainObj, defaultObj) {
```

Comment:
`object.defaults` actually handles null-ish properly, unlike lodash's assign & defaults.  You don't need to filter first.